.cirrus.yml

```diff
@@ -76,7 +76,10 @@ doccheck_task:
 
 osx_task:
     # Run for regular PRs and those with [CI:BUILD] but not [CI:DOCS]
+    # Skip entirely for release-branches (including PRs for them).
     only_if: &not_docs_multiarch >-
```

Comment:
There already is a `not_docs_or_release_branch`, with exactly the right contents if my tired eyes don’t deceive me.

So would it make sense to use that one, and keep the current `cross_task` condition?

Reply:
Durr, yep, my bad.  Good catch.